python/arcticdb/version_store/_store.py

```diff
@@ -2192,34 +2192,7 @@ def lib_cfg(self):
     def open_mode(self):
         return self._open_mode
 
-    def get_info(self, symbol: str, as_of: Optional[VersionQueryInput] = None) -> Dict[str, Any]:
-        """
-        Returns descriptive data for `symbol`.
-
-        Parameters
-        ----------
-        symbol : `str`
-            symbol name
-        as_of : `Optional[VersionQueryInput]`, default=None
-            See documentation of `read` method for more details.
-
-        Returns
-        -------
-        `Dict[str, Any]`
-            Dictionary containing the following fields:
-
-            - col_names, `Dict`
-            - dtype, `List`
-            - rows, `int`
-            - last_update, `datetime`
-            - input_type, `str`
-            - index_type, `index_type`
-            - normalization_metadata,
-            - type, `str`
-            - date_range, `tuple`
-        """
-        version_query = self._get_version_query(as_of)
-        vit, desc = self.version_store.read_descriptor(symbol, version_query)
+    def _process_info(self, symbol: str, vit, desc, as_of: Optional[VersionQueryInput] = None) -> Dict[str, Any]:
```

Comment:
The `vit` was a poor variable name. Converting it to a parameter without type annotation makes it even more difficult to know what it represents.

When changing old code, we should improve on bad practices where possible.

Reply:
there are literally 400 references to fit in the code...

Reply:
Past bad naming is technical debt; however, continuing to use it is just sad....